www/js/app.js

```diff
@@ -333,15 +333,15 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             document.getElementById('articleListWithHeader').style.display = 'none';
         }
     });
-    $("#btnRandomArticle").on("click", function() {
-        $('#prefix').val("");
+    $('#btnRandomArticle').on('click', function() {
```

Comment:
Are you leaving this jQuery function to another PR?

Reply:
Yes, this and all other `.on`  functions where I will need to use `eventListeners`, I shall cover them in a different PR.